client/me/concierge/index.js

```diff
@@ -12,15 +12,12 @@ import page from 'page';
 import controller from './controller';
 import { makeLayout, render as clientRender } from 'controller';
 import { siteSelection, sites } from 'my-sites/controller';
-import reducer from 'state/concierge/reducer';
 
 const redirectToBooking = context => {
 	page.redirect( `/me/concierge/${ context.params.siteSlug }/book` );
 };
 
-export default async ( _, addReducer ) => {
-	await addReducer( [ 'concierge' ], reducer );
```

Comment:
Please keep the `concierge` reducer async-loaded. It is very heavy, as it includes the `moment-timezone` library.

Is any existing data in `concierge` being used in the NPS popup now? Or are you just adding new data about the session count?

Reply:
If the only new thing is the `conciergeSessionCount` field, then that could be moved into the `state/ui/nps-survey-notice`, the NPS modal's own reducer.